.flaskenv

```diff
@@ -1,2 +1,2 @@
-FLASK_ENV=development
+FLASK_DEBUG=True
```

Comment:
FLASK_DEBUG=1

Reply:
See https://github.com/pallets/flask/issues/4714 PR description for usage.

Reply:
Both work: https://github.com/pallets/flask/blob/main/src/flask/helpers.py#L28 but I can of course set to an int

Reply:
Done https://github.com/alerta/alerta/pull/1824/commits/1f74ee217e2eab282afa4941bc095ae348895106 :)